data/transactions/logic/eval_test.go

```diff
@@ -2781,18 +2803,18 @@ bytecblock 0x01234576 0xababcdcd 0xf000baad
 done:
 int 1`, v)
 			require.NoError(t, err)
-			//t.Log(hex.EncodeToString(ops.Program))
+			//t.Log(hex.EncodeToString(ops.Program))                  brhilo
```

Comment:
what is it?)

Reply:
I'm labelling the branch (br) high byte (hi) and low byte (lo) in the string below this line.  I moved it to a comment on a line on its own, and explained.  I'm labeling it like this because this test explicitly clobbering those bytes to test what happens so it was useful for me "mark" them.